pdfminer/converter.py

```diff
@@ -481,7 +481,9 @@ def close(self):
 # XMLConverter
 ##
 class XMLConverter(PDFConverter):
-
+    # Strings are added here as render() runs,
+    # and they get written from render_result at close()
+    render_temp = []
```

Comment:
This should probably be a part of the `__init__` function, altough it does not solve the issue with tests failing when running all of them together.